### PR TITLE
Adding guide for installing latest standalone ui synced with epinio

### PR DESCRIPTION
**Pull request by mmartin24**
Linked to issue in epinio/epinio repo [#1540](https://github.com/epinio/epinio/issues/1540#issuecomment-1151029047)

The purpose is to add a guide that provide a standalone UI with the latests visual updates in synced with the ui-backend and epinio.

The repos involved for syncing are:
- [rancher/dashboard](https://github.com/rancher/dashboard/tree/epinio-dev)
- [epinio/ui-backend](https://github.com/epinio/ui-backend)
- [epinio/epinio](https://github.com/epinio/epinio)

Fix https://github.com/epinio/epinio/issues/1540

### CODE REVIEW

File: docs/developer/install-latest-standalone-ui.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+Go to `/src/jetstream/` and ensure the following line is enabled in `config.properties` file:
+`AUTH_ENDPOINT_TYPE=epinio `
+
+Execute the following comand and copy the id displayed:
+`epinio settings show`
+
+Execute the following with the copied id:
+```
+go build && EPINIO_API_URL=https:${COPIED_ID}.omg.howdoi.website EPINIO_WSS_URL=https://epinio.${COPIED_ID}.omg.howdoi.website EPINIO_API_SKIP_SSL=true EPINIO_VERSION=dev ./jetstream
```
Comment (juadk) [inline docs/developer/install-latest-standalone-ui.md:59]: `EPINIO_API_URL=https:${COPIED_ID}.omg.howdoi.website`
`https:` only? typo or is it correct?

File: docs/developer/install-latest-standalone-ui.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+Execute:
+
+```
+cp src/jetstream/config.example src/jetstream/config.properties 
+```
+
+Go to `/src/jetstream/` and ensure the following line is enabled in `config.properties` file:
+`AUTH_ENDPOINT_TYPE=epinio `
+
+Execute the following comand and copy the id displayed:
```
[the remark this answers is not in the document]
Comment (mmartin24) [reply docs/developer/install-latest-standalone-ui.md:54]: IP: 192.168.121.130
image
My bad

**Comment by mmartin24** [comment]
> I tried it two times from scratch with k3s and I got: image
> 
> ```
> INFO[Thu Jun  9 17:40:24 CEST 2022] Starting HTTPS Server at address: :5443      
> Request: [2022-06-09T17:43:47+02:00] Remote-IP:"127.0.0.1" Method:"GET" Path:"/pp/v1/epinio/rancher/v1/management.cattle.io.setting" Status:200 Latency:119.144µs Bytes-In:0 Bytes-Out:1440
> ERRO[Thu Jun  9 17:43:47 CEST 2022] User session could not be found: Session value not found user_id 
> {"time":"2022-06-09T17:43:47.645751617+02:00","level":"ERROR","prefix":"echo","file":"main.go","line":"1255","message":"code=401, message={\"status\":\"error\",\"error\":\"User session could not be found\"}"}
> Request: [2022-06-09T17:43:47+02:00] Remote-IP:"127.0.0.1" Method:"GET" Path:"/pp/v1/epinio/rancher/v3/users" Status:401 Latency:111.76µs Bytes-In:0 Bytes-Out:60
> ERRO[Thu Jun  9 17:43:47 CEST 2022] User session could not be found: Session value not found user_id 
> {"time":"2022-06-09T17:43:47.65498462+02:00","level":"ERROR","prefix":"echo","file":"main.go","line":"1255","message":"code=401, message={\"status\":\"error\",\"error\":\"User session could not be found\"}"}
> Request: [2022-06-09T17:43:47+02:00] Remote-IP:"127.0.0.1" Method:"POST" Path:"/pp/v1/epinio/rancher/v3/tokens" Status:401 Latency:96.201µs Bytes-In:2 Bytes-Out:60
> ERRO[Thu Jun  9 17:43:47 CEST 2022] User session could not be found: Session value not found user_id 
> {"time":"2022-06-09T17:43:47.65964592+02:00","level":"ERROR","prefix":"echo","file":"main.go","line":"1255","message":"code=401, message={\"status\":\"error\",\"error\":\"User session could not be found\"}"}
> Request: [2022-06-09T17:43:47+02:00] Remote-IP:"127.0.0.1" Method:"GET" Path:"/pp/v1/epinio/rancher/v3/principals" Status:401 Latency:131.607µs Bytes-In:0 Bytes-Out:60
> Request: [2022-06-09T17:43:47+02:00] Remote-IP:"127.0.0.1" Method:"GET" Path:"/pp/v1/epinio/rancher/v3-public/authProviders" Status:200 Latency:77.615µs Bytes-In:0 Bytes-Out:437
> Request: [2022-06-09T17:43:47+02:00] Remote-IP:"127.0.0.1" Method:"GET" Path:"/pp/v1/epinio/rancher/v1/management.cattle.io.setting" Status:200 Latency:42.169µs Bytes-In:0 Bytes-Out:1440
> ERRO[Thu Jun  9 17:43:55 CEST 2022] Error verify epinio creds - response: &{404 Not Found 404 HTTP/1.1 1 1 map[Content-Length:[19] Content-Type:[text/plain; charset=utf-8] Date:[Thu, 09 Jun 2022 15:43:55 GMT] X-Content-Type-Options:[nosniff]] 0xc000062ba0 19 [] false false map[] 0xc00016ce00 0xc0000f02c0}, error: <nil> 
> Request: [2022-06-09T17:43:55+02:00] Remote-IP:"127.0.0.1" Method:"POST" Path:"/pp/v1/epinio/rancher/v3-public/authProviders/local/login" Status:401 Latency:931.486398ms Bytes-In:93 Bytes-Out:150
> ```
> 
> ```
> Platform: k3s
> Kubernetes Version: v1.23.6+k3s1
> Epinio Server Version: v0.8.0-132-g0b489980
> Epinio Client Version: v0.8.0-132-g0b489980
> ```
> 
> Epinio was installed from the helm chart submodule so I should be good with versions.
I

@juadk  I am not sure what happened. It still works for me:
image

```
✔️  Epinio Environment
Platform: k3s
Kubernetes Version: v1.20.11+k3s2
Epinio Server Version: v0.8.0-132-g0b489980
Epinio Client Version: v0.8.0-132-g0b489980

Done preparing k3d environment!
```
I have a working environment if you want to pair to try to see what may be happening.

**Review by juadk — APPROVED** [review]
All good to me with the latest changes.

**Pull request merged**